ocs_ci/ocs/resources/storage_cluster.py

```diff
@@ -371,6 +374,23 @@ def add_capacity(osd_size_capacity_requested):
     storageDeviceSets->count = (capacity reqested / osd capacity ) + existing count storageDeviceSets
 
     """
+    lvpresent = check_local_volume()
+    if lvpresent:
+        path = os.path.join(constants.EXTERNAL_DIR, "device-by-id-ocp")
+        utils.clone_repo(constants.OCP_QE_DEVICEPATH_REPO, path)
+        os.chdir(path)
+        utils.run_cmd("ansible-playbook devices_by_id.yml")
+        dev_paths = utils.run_cmd(f"cat local-storage-block.yaml | grep {osd_size_capacity_requested}")
+        lvcr = get_local_volume_cr()
+        param = f"""[{{ "op": "replace", "path": "/spec/storageClassDevices/0/devicePaths",
+                        "value": {dev_paths}}}]"""
+        lvcr.patch(
+            resource_name=lvcr.get()['items'][0]['metadata']['name'],
+            params=param.strip('\n'),
+            format_type='json'
+        )
+        os.chdir(constants.TOP_DIR)
+        shutil.rmtree(path)
```

Comment:
I'd like for this to be in a utility function that can either provide a list of device paths or the full yaml file we use to create the local volume with. This way we can use it in our deployment quite easily. This doesn't need to be taken up in this PR if it's the only thing holding it up. We can always open another issue or do this as a part of the work to support other platforms like vmware.

Reply:
I'll do that in a follow up PR :)